benchmarks/dynamo/common.py

```diff
@@ -272,7 +272,7 @@ def print_summary_table(data, print_dataframe=False):
                 print(col.ljust(width), f"mean={data[col].mean():.3f}x")
             elif col in ("accuracy"):
                 pass_rate = (data[col] == "pass").mean()
-                print(col.ljust(width), f"pass_rate={100*pass_rate:.2f}%")
+                print(col.ljust(width), f"pass_rate={pass_rate:.3%}")
```

Comment:
Why change the number of digits after the decimal point?

Reply:
Since it's just a print statement I didn't make much of it, but good to verify if this intentional.